ietf/nomcom/utils.py

```diff
@@ -678,6 +678,16 @@ def three_of_five_eligible_8713(previous_five, queryset=None):
         queryset = Person.objects.all()
     return queryset.filter(meetingregistration__meeting__in=list(previous_five),meetingregistration__attended=True).annotate(mtg_count=Count('meetingregistration')).filter(mtg_count__gte=3)
 
+def participants_for_meeting(meeting):
+    """ Return a tuple (checked_in, attended)
+        checked_in = set of onsite, checkedin participants (Person.pk)
+        attended = set of remote participants who attended a session (Person.pk)
+    """
+    checked_in = meeting.meetingregistration_set.filter(reg_type='onsite', checkedin=True).values_list('person', flat=True)
+    sessions = meeting.session_set.filter(Q(type='plenary') | Q(group__type__in=['wg', 'rg']))
+    attended = Attended.objects.filter(session__in=sessions).values_list('person', flat=True)
+    return (set(checked_in), set(attended))
```

Comment:
evaluating these to sets here is not right - we should return querysets and let whatever called it chose when to evaluate. (I also agree this should move to meeting)

Reply:
Moved function and changed to return querysets.